tools/buildsys/src/manifest.rs

```diff
@@ -821,6 +826,8 @@ pub enum ImageFeature {
 
 const EXPERIMENTAL_IMAGE_FEATURES: [&ImageFeature; 0] = [];
 
+const DEPRECATED_IMAGE_FEATURES: [&ImageFeature; 2] = [];
```

Comment:
You can avoid having to specify the size of the array by making this a const reference:

```suggestion
const DEPRECATED_IMAGE_FEATURES: &[&ImageFeature] = [];
```